dask/dataframe/methods.py

```diff
@@ -135,6 +135,13 @@ def wrap_skew_reduction(array_skew, index):
     return array_skew
 
 
+def wrap_kurtosis_reduction(array_kurtosis, index):
+    if isinstance(array_kurtosis, np.ndarray) or isinstance(array_kurtosis, list):
```

Comment:
does this need a dispatch for cupy arrays separately from numpy arrays? cc @quasiben

Reply:
this should be using (is_arraylike](https://github.com/dask/dask/blob/850472dcf887bf1f1fefedf78c75f588ae65d53e/dask/utils.py#L1111)

Reply:
After the `is_arraylike` call, how would it appropriately dispatch?

Reply:
Only getting back to this now. Should I make this change?

Reply:
I think what you have is ok. Just for context Ben this array is going into `pd.Series`. So we probably don't want to use `is_arraylike`.

That said, it would be good if we could do the same thing with CuPy and cuDF. Not sure if we have utility functions to smooth over these things

cc @rjzamora @pentschev (in case either of you know 🙂)

Reply:
Yeah, we don't have anything similar to `__array_function__` currently for dataframes, so the way this is written should be fine now, otherwise we would need to special case things for cuDF here which doesn't make sense unless we do it all over Dask.